eth/peer.go

```diff
@@ -184,6 +184,12 @@ func (p *peer) SendBlockBodies(bodies []*blockBody) error {
 	return p2p.Send(p.rw, BlockBodiesMsg, blockBodiesData(bodies))
 }
 
+// SendBlockBodiesRLP sends a batch of block contents to the remote peer from
+// an already RLP encoded format.
+func (p *peer) SendBlockBodiesRLP(bodies []*blockBodyRLP) error {
+	return p2p.Send(p.rw, BlockBodiesMsg, blockBodiesRLPData(bodies))
```

Comment:
Please use `rlp.RawValue` instead of `blockBodyRLP` when #1778 is on develop.
The conversion to `blockBodiesRLPData` is also unnessary (package rlp knows how to encode slices of encoder types).